cmake/git_version.cmake

```diff
@@ -0,0 +1,29 @@
+#    The MIT License (MIT)
```

Comment:
the latest version of this in cheetah should fix a bug that appears in git headless mode